docs/installation.md

```diff
@@ -42,7 +42,29 @@ $ npm install -D MithrilJS/mithril.d.ts#next
 
 ---
 
-### Quick start with Webpack
+### Create a project locally
+
+You can use one of several existing Mithril starter templates such as
+* [mithril-esbuild-starter](https://github.com/kevinfiol/mithril-esbuild-starter)
+* [mithril-vite-starter](https://github.com/ArthurClemens/mithril-vite-starter)
+* [mithril-rollup-starter](https://github.com/kevinfiol/mithril-rollup-starter)
+
+For example, if you'd like to get started with `mithril-esbuild-starter`, run the following commands:
+```bash
+# Clone the the template to a directory of your choice
+npx degit kevinfiol/mithril-esbuild-starter hello-world
+
+# Navigate to newly scaffolded project
+cd ./hello-world/
+
+# Install dependencies
+npm install
+
+# Build the app and watch for changes
+npm run dev
```

Comment:
Seems like port 8080 is the default. Maybe a hint would be nice, if port 8080 is already in use (in my case traefik).

<img width="478" alt="port-8080-already-in-use" src="https://github.com/MithrilJS/mithril.js/assets/1334161/c794aa37-974c-460f-bd6c-e636a57d9284">

Reply:
I updated the port used by the esbuild template, and the dev script should log the port to the console by default.